src/features/user/ManagePayouts/screens/EditBankAccount.tsx

```diff
@@ -105,9 +109,20 @@ const EditBankAccount = (): ReactElement | null => {
       if (foundAccount) {
         setAccountToEdit(foundAccount);
       } else {
-        handleError({
-          message: t('errors.noAccountFound'),
-        });
+        if (errors) {
+          setErrors([
+            ...errors,
+            {
+              message: t('errors.noAccountFound'),
+            },
+          ]);
+        } else {
+          setErrors([
+            {
+              message: t('errors.noAccountFound'),
+            },
+          ]);
+        }
```

Comment:
Setting errors to null makes sense since, after dismissing the errors (either by timeout or user intervention), there won't be anything in the errors state. 

Moreover, if we keep an empty array, then we would need to check the errors' length, which can be prone to errors or bugs.